app/controllers/application_controller/compare.rb

```diff
@@ -748,44 +748,47 @@ def compare_init(mode)
     @exists_mode = session[:miq_exists_mode]
   end
 
-  # Compare selected VMs
+  # Compare selected items
   def comparemiq
     assert_privileges(params[:pressed])
-    vms = []
+    items = []
     if !session[:checked_items].nil? && @lastaction == "set_checked_items"
-      vms = session[:checked_items]
+      items = session[:checked_items]
     else
-      vms = find_checked_items
+      items = find_checked_items
     end
```

Comment:
or maybe just leave it for now. we are here to fix things not for refactoring at this point